settings/OpenExceptionalForm/ExceptionWrapper.js

```diff
@@ -417,68 +341,76 @@ class ExceptionWrapper extends React.Component {
     });
   }
 
-  setServicePoints(sps) {
+  setServicePoints = sps => {
     this.setState({
       servicePoints: sps,
     });
     this.setEvents(sps);
   }
 
-  separateEvents(event) {
+  separateEvents = event => {
     const temp = [];
+    const {
+      start,
+      end,
+      openingDays,
+      exceptional,
+      id,
+    } = event;
     let g = 0;
-    for (let i = 0; i < moment(event.end)
-      .diff(moment(event.start), 'days') + 1; i++) {
-      const today = moment(event.start)
+
+    for (let i = 0; i < moment(end)
+      .diff(moment(start), 'days') + 1; i++) {
+      const today = moment(start)
         .add(i, 'days')
         .format('dddd')
         .toUpperCase();
 
       const dates = [];
       let eventContent;
 
-      for (let j = 0; j < event.openingDays.length; j++) {
-        if (event.exceptional === false) {
-          if (today === event.openingDays[j].weekdays.day) {
-            if (event.openingDays[j].openingDay.allDay === true) {
+      for (let j = 0; j < openingDays.length; j++) {
+        if (exceptional === false) {
+          if (today === openingDays[j].weekdays.day) {
+            if (openingDays[j].openingDay.allDay === true) {
               dates.push(
                 <div>All day</div>
               );
             }
-            for (let k = 0; k < event.openingDays[j].openingDay.openingHour.length; k++) {
+            for (let k = 0; k < openingDays[j].openingDay.openingHour.length; k++) {
```

Comment:
This is not a change request. But maybe it will be nice to save, for example, **openingHour** to the new const to prevent using long paths like this one **openingDays[j].openingDay.openingHour** in a lot of places.